assets/rln-relay/rln-message-verification.msc

```diff
@@ -0,0 +1,45 @@
+# MSC for some fictional process
```

Comment:
Ditto other.

I also notice the picture is quite hard to read (have to zoom in), how does it render in the markdown spec? If it is too hard to read, maybe we can make it take less horizontal space somehow?

Reply:
Sure, will find a way to shrink the width